libobs/obs-source.c

```diff
@@ -5624,14 +5633,26 @@ void obs_source_set_monitoring_type(obs_source_t *source,
 	signal_handler_signal(source->context.signals, "audio_monitoring",
 			      &data);
 
+	uint32_t flags = obs_source_get_output_flags(source);
+	bool master = (flags & OBS_SOURCE_AUDIO_TRACK) != 0;
```

Comment:
If removing the new flag as my suggestion on other locations, this line should be updated.
```suggestion
	bool master = obs_source_is_audio_track(source);
```